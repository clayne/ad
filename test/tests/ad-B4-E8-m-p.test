ad | -B4 -E8 -mp | pjl-conductor-200.jpg | 0
