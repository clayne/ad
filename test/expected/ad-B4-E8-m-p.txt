0000000000000010: 0000 0008 000F 0100 0003 0000 0001 00C8  ................
00000000000009A0: 7874 4973 4854 4D4C 626F 6F6C 0100 0000  xtIsHTMLbool....
00000000000009B0: 0863 656C 6C54 6578 7454 4558 5400 0000  .cellTextTEXT...
